src/routes/canvas/toolbar/tools/Eraser.tsx

```diff
@@ -1,15 +1,16 @@
 import React from "react";
 import { InkingTool } from "@microsoft/live-share-canvas";
 import { Button } from "@fluentui/react-components";
+import { Eraser24Regular, EraserSegment24Regular} from "@fluentui/react-icons"
 
 import Tool, { ToolProps } from "./Tool";
-import SizePicker from "./SizePicker";
+import MySizePicker from "./MySizePicker";
```

Comment:
Not a big fan of the name. If a more appropriate name can be used would be good, but it's just a minor thing

Reply:
How about  SizeSelector / SizeSlider?

Reply:
Sure, sounds good!